examples/basic/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "aws" {
+  region = "eu-west-2"
+
+  default_tags {
+    tags = {
+      Terraform = true
+    }
+  }
+}
+
+module "data_qa" {
+  source                     = "../../terraform"
+  cloudfront_allowed_subnets = ["213.238.187.134/32"]
```

Comment:
hardcode?

Reply:
Going to use 255.255.255.255/32 as a placeholder